docs/data-ai/ai/train.md

```diff
@@ -1,9 +1,601 @@
 ---
 linkTitle: "Train other models"
 title: "Train other models"
+tags: ["data management", "ml", "model training"]
 weight: 30
 layout: "docs"
 type: "docs"
-no_list: true
-description: "TODO"
+aliases:
+  - /services/ml/upload-training-script/
+  - /how-tos/create-custom-training-scripts/
+languages: ["python"]
+viamresources: ["mlmodel", "data_manager"]
+platformarea: ["ml"]
+description: "If you want to train models to custom specifications, write a custom training script and upload it to the Viam Registry."
+date: "2024-12-04"
 ---
+
+You can create custom Python training scripts that train ML models to your specifications using PyTorch, Tensorflow, TFLite, ONNX, or any other Machine Learning framework.
```

Comment:
you should probably also include that you could use training scripts that are in the registry already